### DISCUSSION TITLE
Mesen Core? (NES/FC)

### DISCUSSION THREAD

**Discussion by J7NYS** [OnionUI/Onion#484] [Q&A]
Hello Onioneers

I was just wondering if anyone knew how to add Mesen core (NES/FC) to the Miyoo Mini? Is there a way to do? It's easy of course to add cores to Retroarch with other builds but if I am presuming correctly onion os only has the two NES cores available? If anyone is wondering why, I wan't to try out a particular demo (Cross Paint) which will not run on the included cores. It's not a probalem if not, I just thought it would be fun to see the project running on a small device.

Thanks in advance if anyone can help,
Jonny

**Comment by schmurtzm** [comment]
Hi, I have compiled different Messen core from different repo (including the official one) but Messen is always too slow for the Miyoo Mini : slow and with a lot of sound stuttering. 
If you still want to test the core you can find it here : https://github.com/schmurtzm/Miyoo-Mini-Retroarch-builder/actions
